packages/liveblocks-react/src/__tests__/useThreads.test.tsx

```diff
@@ -492,6 +521,179 @@ describe("useThreads", () => {
 
     unmount();
   });
+
+  test("should sort threads by creation date before returning", async () => {
+    const oldThread = dummyThreadData();
+    oldThread.createdAt = new Date("2021-01-01T00:00:00Z");
+
+    const newThread = dummyThreadData();
+    newThread.createdAt = new Date("2021-01-02T00:00:00Z");
+
+    server.use(
+      mockGetThreads(async (_req, res, ctx) => {
+        return res(
+          ctx.json({
+            data: [newThread, oldThread], // The order is intentionally reversed to test if the hook sorts the threads by creation date
```

Comment:
I understand that the server returns the thread already sorted by creation date, so this test doesn't replicate server behavior correctly but attempts to test the sorting behavior on client side.